src/Sylius/Behat/Context/Ui/Admin/Helper/EditToShowPageSwitcherTrait.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Ui\Admin\Helper;
+
+trait EditToShowPageSwitcherTrait
+{
+    private string $resourceName;
+
+    public function switchToShowPage(): void
```

Comment:
I think it would be better to create one `NavigationTrait` class with these three methods. Additionally, include a method `getResourceName` that returns a string instead of the `defineResourceName` method.